### PR TITLE
Update ClinVar import (e107)

### PR DESCRIPTION
**Pull request by dglemos**
- Update version (ENSVAR-4439)
- Fix sql query to remove entries from allele_synonym and also remove entries from variation_citation (ENSVAR-4442)
- Update post process clinvar script (ENSVAR-4555)

### CODE REVIEW

**Review by jamie-m-a — APPROVED** [review]

File: scripts/import/import_clinvar_xml [hunk 1]
```diff
@@ -1441,7 +1443,23 @@ sub clean_old_data{
...
+  # first check which entries to remove
+  my $var_syn_sel_sth    = $dba->dbc()->prepare(qq[ select allele_synonym_id from allele_synonym where variation_id in
+                                                    (select variation_id from variation v, source s
+                                                    where v.source_id = s.source_id and s.name ='ClinVar')
+                                                ]);
+  $var_syn_sel_sth->execute();
+  my $data = $var_syn_sel_sth->fetchall_arrayref();
+  # remove entries from allele_synonym
+  foreach my $allele_syn (@{$data}){
+    my $allele_syn_id = $allele_syn->[0];
```
Comment (sarahhunt) [inline scripts/import/import_clinvar_xml:1456]: we don't need to refine the variable here

File: scripts/import/import_clinvar_xml [hunk 2]
```diff
@@ -1441,7 +1443,23 @@ sub clean_old_data{
...
+    my $allele_syn_delete_sth = $dba->dbc()->prepare(qq[ delete from allele_synonym where allele_synonym_id = $allele_syn_id ]);
+    $allele_syn_delete_sth->execute();
```
Comment (sarahhunt) [inline scripts/import/import_clinvar_xml:1458]: It would be good to capture fails and die here, though the same goes for all the clean up steps.

Comment (sarahhunt) [inline scripts/import/import_clinvar_xml:1458]: It would be good to capture fails and die here, though the same goes for all the clean up steps.

**Review by sarahhunt — COMMENTED** [review]
There are a couple of minor improvements possible here

File: scripts/import/import_clinvar_xml (hunk 1 above)
Comment (dglemos) [reply scripts/import/import_clinvar_xml:1456]: You mean we can use $allele_syn->[0] instead of $allele_syn_id?

**Comment by sarahhunt** [comment]
merged to master & release/107

**Pull request merged**